server/routes/handlers.js

```diff
@@ -11,14 +11,22 @@ const handleExperienceFullfilment = item => (agentItem) => {
 };
 
 const handleEducationFullfilment = item => (agentItem) => {
-  const companyName = agentItem.parameters['education-degree'];
-  const response = getResponse('educations')(companyName, item);
+  const degreeName = agentItem.parameters['education-degree'];
+  const response = getResponse('educations')(degreeName, item);
 
-  saveDemand(companyName);
```

Comment:
Maybe in the future, I have to improve the way I collect demand. But for the moment it is ok